app/views/maps/_sidebar_exports.html.erb

```diff
@@ -112,6 +115,26 @@ var exporting, exportResolutionSlider
     setInterval(exportProgress,5000)
```

Comment:
Hi! Could we refactor this section to rely on. The new Json output? We don't need to download the file, but this will standardize how we fetch it. Thanks!

Reply:
Ok will do!